streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -440,41 +402,35 @@ boolean tryToCompleteRestoration() {
      * @throws TaskMigratedException if the task producer got fenced (EOS only)
      */
     void handleRevocation(final Collection<TopicPartition> revokedPartitions) {
-        final Set<TopicPartition> remainingPartitions = new HashSet<>(revokedPartitions);
+        final Set<TopicPartition> remainingRevokedPartitions = new HashSet<>(revokedPartitions);
```

Comment:
nit: Add in the above javadoc that we should only revoke active tasks here?

Reply:
ack